I have separated the JS and CSS of my code and referenced them for mo…

metehansalih06 opened the pull request:
…re readability. I also tried to link the fontawesome as a css file but it doesnt work for me. I will continue finding a solution for that tomorrow.

The Icons are not shown yet, because the browser cant find the file for the icons. I will find a solution for that tomorrow hopefully. I will also add the database data, so that the site will be finished.

metehansalih06 committed:
I have separated the JS and CSS of my code and referenced them for more readability. I also tried to link the fontawesome as a css file but it doesnt work for me. I will continue finding a solution for that tomorrow.

The Icons are not shown yet, because the browser cant find the file for the icons. I will find a solution for that tomorrow hopefully. I will also add the database data, so that the site will be finished.

Co-Authored-By: Mohamed Amin Ben Chrifa <mohamed6aminbenchrifa@gmail.com>

ookii-tsuki commented on app/order/templates/{{ url_for('static', filename='css/fontawesome.css') }}:
Please delete this file and its parent folder

ookii-tsuki commented on app/order/templates/past_orders.html:
```diff
@@ -7,291 +7,11 @@
 
     <!-- Font Awesome für die Icons -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link href="{{ url_for('static', filename='css/fontawesome.css') }}" rel="stylesheet">
```
The reason the icons don't show up anymore because you're missing a font file.
The font files you need are `fa-solid-900.ttf` and `fa-solid-900.woff2` which you can find [here](https://github.com/FortAwesome/font-awesome-sass/tree/master/assets/fonts/font-awesome).

Download those two and put both of them in `/static/webfonts` folder in the project.

ookii-tsuki reviewed (commented):
Everything else looks good for now.

If you're reading this from the email, open the review on the github website to be able to see the references.

metehansalih06 committed:
I have finished the past_orders page and added the fonts for fontawesome. Also deleted the folder fontawesome. The data fetching seems not to work for now, I tried logging in but still after logging in, the error is shown that the user must be logged in.

I will work on the search page for now, so that I can navigate directly to the past orders page. Maybe it will work then. If not, I will try to find a solution for that.

Co-Authored-By: Mohamed Amin Ben Chrifa <mohamed6aminbenchrifa@gmail.com>

metehansalih06 committed:
the back button now directs to the main page

the page still looks like it doesnt fetch data from the databse, I try finding a solution for that

ookii-tsuki commented on app/static/js/order/past_orders.js:
```diff
@@ -0,0 +1,135 @@
+                method: 'GET',
+                headers: {
+                    'Authorization': `Bearer ${token}`
+                }
```
you don't need a Bearer token. The login session logic is in the server so remove that and replace the headers sections with this
```json
headers: {
'Content-Type': 'application/json',
}
```
it will work then

ookii-tsuki commented on app/static/css/order/past_orders.css:
```diff
@@ -0,0 +1,281 @@
+
+    /* Foto Layout */
+    .photo {
+        width: 90px; 
```
set this to `50px` or something to make the logo smaller

metehansalih06 committed:
I changed the size of the logo and changed the bearer-token part. I dont know if it works now, it needs to be checked with orders.

I will now continue working on the search page.

ookii-tsuki reviewed (commented):
lgtm 👍. fix the merge conflicts so that we can merge to main.

an easy way to do it is to update the branch from main. when you do that you'll be prompted in Github Desktop to solve the merge conflicts. you can choose to open them in VS Code one by one if many and choose whether to keep your changes (`HEAD`) or the incoming changes.

ookii-tsuki commented:
Btw can you add an option to logout?
like this:
![Screenshot_20250117_205916](https://github.com/user-attachments/assets/8eb78c26-798d-4248-9764-8a666e7b990f)

metehansalih06 committed:
Merge branch 'main' into metehan/Lieferspatz-Hauptseite

metehansalih06 committed:
wallet is shown correctly on the page

ausloggen button is added and works correctly.

metehansalih06 committed:
I changed the page route so that the page loads

metehansalih06 committed:
I changed the format of the wallet

It shows 2 digits after the comma

metehansalih06 committed:
Changed the text of the notification

It will now convert the status number to the according string.

metehansalih06 committed:
Fixed the red dot on the bell button

The red dot should be at the right place now and disappears once the button is clicked.

metehansalih06 committed:
I updated the layout of the past order items

Everything should look like intended now
